mountinfo/mounted_linux.go

```diff
@@ -7,6 +7,35 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+// MountedFast is a method of detecting a mount point without reading
+// mountinfo from procfs. A caller can only trust the result if no error
+// and sure == true are returned. Otherwise, other methods (e.g. parsing
+// /proc/mounts) have to be used. If unsure, use Mounted instead (which
+// uses MountedFast, but falls back to parsing mountinfo if needed).
+//
+// If a non-existent path is specified, an appropriate error (rather than
+// "not mounted") is returned. In case the caller is not interested in this
+// particular error, it should be handled separately using e.g.
+// errors.Is(err, os.ErrNotExist).
+//
+// This function is only available on Linux. The implementation mostly
+// relies on openat2(2) syscall, available since Linux kernel v5.6.
```

Comment:
nit: this reads to me that we rely on openat2. In reality, we chose the sys call based on the linux version. Would you be able to update the language bit. 

The implementation relies on openat2(2) sys call when available. When it is not, it will use stat(2) syscall which can detect normal (but not bind) mounts.

Reply:
I'm not sure how to spell it out so the kernel version information is not lost -- and I think it makes sense to mention it.

Ended up with

```go
// This function is only available on Linux. When available (since kernel
// v5.6), openat2(2) syscall is used to reliably detect all mounts. Otherwise,
// the implementation falls back to using stat(2), which can reliably detect
// normal (but not bind) mounts.
```

Reply:
This sounds much better to me. Thanks